ACE/ace/CDR_Stream.h

```diff
@@ -791,6 +807,18 @@ class ACE_Export ACE_InputCDR
     ACE_CDR::Octet &ref_;
   };
 
+  struct ACE_Export to_int8
```

Comment:
Is this really required, isn't int8_t a distinct type itself?

Reply:
`(u)int*_t` are typedefs, probably at least one is the same as the one used for ACE_CDR's Octet, etc.